imaging/ml/ml_codelab/README.md

```diff
@@ -18,33 +18,20 @@ PROJECT_NUMBER=`gcloud projects describe ${PROJECT_ID} | grep projectNumber | se
 COMPUTE_ENGINE_SERVICE_ACCOUNT="${PROJECT_NUMBER}-compute@developer.gserviceaccount.com"
 gcloud projects add-iam-policy-binding ${PROJECT_ID} --member "serviceAccount:${COMPUTE_ENGINE_SERVICE_ACCOUNT}" --role roles/pubsub.admin
 ```
-## Create Datalab instance
+## Create AI Platform Notebooks instance
 
-This following shell command will create a SSH tunnel allowing you to visit the Datalab UI in your web browser. You can run this on a shell in your **local machine** or **[Cloud Shell](https://cloud.google.com/shell/docs/)**.
+Follow the steps listed [here](https://cloud.google.com/ai-platform/notebooks/docs/create-new)
```

Comment:
>It would be good to be more specific: e.g. describe the recommended machine type (used to be n1-standard-4)

done
>Also, it would be better to still do this via the command line if possible, since it is overall less disruptive to the workflow.

usually documentation provides description of command line way, but for this example i can't find it